apps/antalmanac/src/components/RightPane/Map/UCIMap.tsx

```diff
@@ -430,7 +430,7 @@ export default class UCIMap extends PureComponent {
             const courses = pins[buildingCode];
             for (let index = courses.length - 1; index >= 0; index--) {
                 const [event, eventIndex] = courses[index];
-                const courseString = `${event.title} ${event.sectionType} @ ${event.bldg}`;
+                const courseString = `${event.title} ${event.sectionType} @ ${event.bldg} : ${event.sectionCode} @ ${event.term}`;
                 if (locationData === undefined) return;
 
```

Comment:
- Could we define `courseString` later? We shouldn't pre-emptively evaluate it and then not use it lol
- Also I think I'd prefer it named as `key`, since that's the only thing it's used for
```suggestion
                if (locationData === undefined) return;
                const key = `${event.title} ${event.sectionType} @ ${event.bldg} : ${event.sectionCode} @ ${event.term}`;
```

Reply:
Done!